version/news/NEWS-2024.12.1-kousa-dogwood.md

```diff
@@ -7,6 +7,7 @@
 - Noto Sans is now used as a fallback proportional font (for RStudio UI elements) on Linux systems. (#15547)
 
 #### Posit Workbench
+- The secret key used to sign authorization cookies in environments where Workbench is configured with SSL can now be set via the `WORKBENCH_ENCRYPTION_KEY` environment variable. (rstudio-pro#7169)
```

Comment:
I think I would find

> used to sign authorization cookies in environments where Workbench is configured with SSL

kind of confusing as a customer or user. Is it possible there is a simpler explanation here, like "used to sign browser cookies"? Or is that not accurate/precise enough?

Reply:
That's a good point, and had crossed my mind as well. I'll simplify with "browser cookie" as suggested!